_pages/alumni.html

```diff
@@ -21,4 +21,5 @@ <h5>Alumni Quotes About Team 1418</h5>
 </main>
 
 <script src="/js/alumni.js"></script>
-<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyABRmbhIjpLDCHh_KrgVpdZdy4Pvs_X2jc&callback=initMap" async defer></script>
+<!-- <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyABRmbhIjpLDCHh_KrgVpdZdy4Pvs_X2jc&callback=initMap" async defer></script> -->
+<script src="https://maps.googleapis.com/maps/api/js?key=&callback=initMap" async defer></script>
```

Comment:
What's this about?

Reply:
That was to run the website on local host, I'll change it back